themes-default/slim/src/index.js

```diff
@@ -144,24 +144,6 @@ if (!pathname.includes('/login') && !pathname.includes('/apibuilder')) {
         MEDUSA.config.loading = '<img src="images/loading16' + MEDUSA.config.themeSpinner + '.gif" height="16" width="16" />';
 
         $(document).ready(UTIL.init);
-
-        MEDUSA.config.indexers.indexerIdToName = indexerId => {
-            if (!indexerId) {
-                return '';
-            }
-            return Object.keys(MEDUSA.config.indexers.config.indexers).filter(indexer => { // eslint-disable-line array-callback-return
-                if (MEDUSA.config.indexers.config.indexers[indexer].id === parseInt(indexerId, 10)) {
-                    return MEDUSA.config.indexers.config.indexers[indexer].name;
-                }
-            })[0];
```

Comment:
Just a small note:
I changed the logic in the new function, but it's actually still returning the same value.
`Object.keys(...).filter(callback)` returns the elements of the array that the callback function returns a truthy value for, **and not the value the function returns.**
Which in this case, was the keys of `MEDUSA.config.indexers.config.indexers`.
That's why the new function does the same thing, but with a simpler logic.
```js
        return Object.keys(indexers).find(name => indexers[name].id === parseInt(indexerId, 10));
```